### ISSUE TITLE
$mdSticky not working with <md-toolbar>

### ISSUE THREAD

**Issue by foxcode3** [angular/material#10456]
**Actual Behavior**:
 -  `What is the issue? *`
`$mdSticky` not working with `<md-toolbar>`
 -  `What is the expected behavior?`
Stick the md-toolbar in top position

**CodePen** (or steps to reproduce the issue): *
 -  `CodePen Demo which shows your issue:`
http://codepen.io/aendran/pen/PpzbNM
 -  `Details:`
When I try to stick the md-toolbar in top position its not working properly. I created a custom directive using $mdSticky service in AngularJS.

**Angular Versions**: *
 -  `Angular Version:`1.6.1
 -  `Angular Material Version:`1.1.3

**Additional Information**:
 -  `Browser Type: *`Chrome
 -  `Browser Version: *`56.0.2924.87 (64-bit)
 -  `OS: *` Windows 8.1
 -  `Stack Traces:`

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.

**Comment by EladBezalel** [comment]
@DevVersion if i recall correctly you were the last one who worked on mdSticky, is that possible?

**Comment by DevVersion** [comment]
Yeah, I had a big proposal for fixing all the clone stuff but I won't find any time to finish it.

**Comment by Splaktar** [comment]
I can reproduce this in an [updated CodePen](https://codepen.io/Splaktar/pen/mdVbPdx?editors=1010) with `1.1.22`.
